packages/cli/src/lib/utils/get-available-piece-names.ts

```diff
@@ -0,0 +1,9 @@
+import { readdir } from 'node:fs/promises';
+
+export const getAvailablePieceNames = async (
```

Comment:
Thiis is outdated, since all pieces now lives in community or custom

Please check the `findAllPiecesDirectory` in tools/script at HEAD